### PR TITLE
Add --with-external-bliss option to unvendor bliss

### PR DESCRIPTION
**Pull request by isuruf**
cc @james-d-mitchell, @kiwifb, @dimpase

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -90,16 +90,25 @@ matrix:
       - CFLAGS="$CFLAGS -I$HOME/miniconda3/include -O2 -g"
       - LDFLAGS="$LDFLAGS -L$HOME/miniconda3/lib"
 
+    - env:
```
Comment (isuruf) [inline .travis.yml]: I added a new travis job here. If you want, I'll combine this with the `external-planarity` job.

**Comment by james-d-mitchell** [comment]
I'm happy with this. Any comments @wilfwilson?

**Comment by dimpase** [comment]
I'd test how this works within Sage, although I guess @isuruf already looked into it.

File: .travis.yml (hunk 1 above)
Comment (wilfwilson) [reply .travis.yml]: I would prefer it is was combined, pleased.

**Comment by wilfwilson** [comment]
@james-d-mitchell Only comment I have is the one I just made about the new Travis job.

**Comment by dimpase** [comment]
Is anyone working on an improved version, `--with-eternal-bliss` ?

**Comment by james-d-mitchell** [comment]
Not at present, but I'd be happy to accept a PR which guaranteed `--with-a-few-hours-of-bliss`

**Comment by isuruf** [comment]
Can somebody restart travis-ci? Tests pass in my fork

**Comment by james-d-mitchell** [comment]
Looks good, can we merge @wilfwilson?

**Pull request merged**